scripts/windows/install_erlang.bat

```diff
@@ -0,0 +1,82 @@
+@echo off
```

Comment:
This script is to streamline installation of erlang supporting multiple simultaneous versions and uses erlang-solutions as mirror (by default) as it is way too faster to download. CI download and install will require a script anyway. Chocolatey script does not support installing multiple versions properly and also leaves a running process which bugs the job execution (waits forever for it to finish)